miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/MiniAppInfo.kt

```diff
@@ -25,6 +25,6 @@ data class MiniAppInfo(
      */
     val icon: String,
 
-    internal val versionId: String,
```

Comment:
This is okay for now. We would want to change it back once we update the backend API sepcs.

Reply:
Yeah, the API spec has changed it bit. There is now a `versionId` and `versionTag`. `versionTag` should be public, but `versionId` should be private.